ui/apps/platform/src/services/ComplianceScanConfigurationService.ts

```diff
@@ -195,9 +195,9 @@ export type ComplianceRunReportResponse = {
  */
 export function runComplianceReport(
     scanConfigId: string,
-    scanNotificationMethod: 'EMAIL' | 'DOWNLOAD'
+    reportNotificationMethod: 'EMAIL' | 'DOWNLOAD'
```

Comment:
`scanNotificationMethod` did not make sense. The following is a more correct description.